lib/screen/claim/select_account_page.dart

```diff
@@ -238,5 +241,13 @@ class _SelectAccountPageState extends State<SelectAccountPage> with RouteAware {
       AppRouter.homePage,
       (route) => false,
     );
+    final token = claimRespone?.token;
+    final caption = claimRespone?.airdropInfo.twitterCaption;
+    if (token == null || caption == null) {
```

Comment:
can we define an empty caption when it's `null`? I don't think we should prevent the claiming flow when this info is empty.